.github/workflows/stable.yml

```diff
@@ -165,7 +165,14 @@ jobs:
         with:
           args: install nasm
 
+      - name: Run cargo build
```

Comment:
This will cause all the crates to be built, and then get re-built immediately after for the tests step. Can we limit this to just `matrix.target == 'x86_64-pc-windows-gnu'`?

Reply:
Sure. Done.